changelogs/unreleased/gh-8657-move-triggers-to-trigger-registry.md

```diff
@@ -0,0 +1,8 @@
+## feature/lua
+
+* **[Breaking change]** Triggers from space and `box.session` were moved to
+  trigger registry (gh-8657). Now triggers set with `space:on_replace()` or
+  `space:before_replace()` are attached to the space id, not the space object.
+  So if you delete a space with registered triggers and create another space
+  with the same id, it will use the triggers that are left from the deleted
+  space.
```

Comment:
```suggestion
* **[Breaking change]** Triggers from a space and `box.session` were moved to the
  trigger registry (gh-8657). Now triggers set with `space:on_replace()` or
  `space:before_replace()` are attached to the space id, not the space object.
  So, if you delete a space with registered triggers and create another space
  with the same id, it will use the triggers that are left from the deleted
  space.
```

Reply:
I think that there must be `Triggers from space and ..` because it means space in general, not a particular space..

Reply:
I wrote "Triggers from `space_object`".

Reply:
Fixed.